config/opendatahub-io/org.yaml

```diff
@@ -134,6 +134,40 @@ orgs:
         privacy: closed
         repos:
           s2i-lab-elyra: maintain
+      repo-odh-model-controller-write:
+        description: "Members with Write access to the odh-model-controller repository"
+        members:
+        - 4n4nd
```

Comment:
We definitely don't want 4n4nd here.

Reply:
From an offline conversation, this new team and it's member are being grant the same access that they already have as members of the `Developers` group. See #4